src/server.ts

```diff
@@ -3,17 +3,13 @@ import {InfluxDB} from 'influx';
 import * as Koa from 'koa';
 import * as bodyParser from 'koa-bodyparser';
 import {Connection} from 'typeorm';
-import {qaRoutes} from '../test/integration/routes';
 import {loadConfig} from './config';
 import MQTTTopicClient from './mqtt/client';
 import {influxConn, pgConn} from './persistence/database';
 import {routes} from './routes';
 import CoffeeDetector from './service/coffeeDetector';
-import DataRecorder from './service/recorder/dataRecorder';
-import {InfluxDataRecorder} from './service/recorder/influxDataRecorder';
-import { gravityTransformerTagged } from './service/thingy';
 
-async function bootstrap(samples: boolean) {
+async function bootstrap() {
```

Comment:
Why did you remove this part? It can be helpful to work with sample data even no tests are executed (backend-frontend development)